scripts/download-e2e-test-config.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -ue
+
+pushd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null
+# shellcheck disable=SC1091
+[[ ${UTIL_SOURCED-no} != yes && -f ./util.sh ]] && source ./util.sh
+popd > /dev/null
+
+DEPLOYMENT_BUCKET="${1}"
+ENV_NAME="${2}"
+
+CONFIG_S3_PATH="s3://${DEPLOYMENT_BUCKET}/e2e-test-configs/cypress.${ENV_NAME}.json"
```

Comment:
Should this script be put on GitHub since it is mostly intended for team use? Or is it intended for customers to be able to use as well? It requires them to setup the config in S3 if so.

Reply:
We want both repos to be identical. No sensitive information, and it also helps parameterize the path now if customers want to use it (previously it wasn't possible)